src/test/java/com/microsoft/aad/adal4j/AdalAccessTokenResponseTest.java

```diff
@@ -21,18 +21,19 @@
 
 import java.text.ParseException;
 
-import org.testng.Assert;
-import org.testng.annotations.Test;
-
 import com.nimbusds.jwt.JWT;
+import com.nimbusds.oauth2.sdk.token.AccessToken;
 import com.nimbusds.oauth2.sdk.token.BearerAccessToken;
 import com.nimbusds.oauth2.sdk.token.RefreshToken;
 import com.nimbusds.oauth2.sdk.util.JSONObjectUtils;
+import com.nimbusds.openid.connect.sdk.token.OIDCTokens;
+import org.testng.Assert;
+import org.testng.annotations.Test;
 
 @Test(groups = { "checkin" })
 public class AdalAccessTokenResponseTest extends AbstractAdalTests {
 
-    private final String idToken = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ik5HVEZ2ZEstZnl0aEV1THdqcHdBSk9NOW4tQSJ9."
+    private final String token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ik5HVEZ2ZEstZnl0aEV1THdqcHdBSk9NOW4tQSJ9."

```

Comment:
any reason to rename this variable? calling it token does not tell me what it is without actually decoding the JWT.